README.md

```diff
@@ -1,3 +1,148 @@
+
+var sendEmailRequest = new SendEmailRequestDetails(
+    providerConfigurationId: configuration.ProviderConfigurationId,
+    recipients: new RecipientEmailDetails(
+            [
+                {
+                    email = " To Email Addres 1"
+                },
+                {
+                    email = "To Email Addres 2"
+                }
+            ],
+            [
+                {
+                    email = " Cc Email Addres 1"
+                },
+                {
+                    email = "Cc Email Addres 2"
+                }
+            ],
+            [
+                {
+                    email = "Bcc Email Addres 1"
+                },
+                {
+                    email = "Bcc Email Addres 2"
+                }
+            ]),
+    body: "Hello, world!",
+    subject: "Email subject",
+    attachment: new AttachmentDetails  [
+                {
+                    attachmentId: new Guid("00000000-0000-0000-0000-000000000000")
+                }
+            ],
+    schedule: null,
+    template: new TemplateData(
+    {
+        templateId: new Guid("00000000-0000-0000-0000-000000000000"),
+        mergeData: {}
+    }
+    ),
+    callbackUrl: "provide your callback Url")
+
+var emailResponse = await emailClient.SendEmail(sendEmailRequest).ConfigureAwait(false);
+
+```
+The simplest example for upload attachment is:
+
+```csharp
+using var httpClient = new HttpClient();
+using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+var emailClient = new EmailClient(httpClient, tokenProviderFactory, configuration.EmailOptions);
+
+ var attachmentRequest = new AttachmentRequestDetails(new Guid(configurationId), attachmentFile);
+result = await emailClient.AddAttachment(attachmentRequest).ConfigureAwait(false)
+```
+
+The simplest example for upload template is:
+
+```csharp
+using var httpClient = new HttpClient();
+using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+var emailClient = new EmailClient(httpClient, tokenProviderFactory, configuration.EmailOptions);
+
+ var attachmentRequest = new TemplateRequestDetails(new Guid(configurationId), templateFile);
+result = await emailClient.AddAttachment(attachmentRequest).ConfigureAwait(false)
+```
+
+## Sample application
+## How to configure the Email client
+
+Perhaps the easiest way to configure the Email client is from Application Settings.
+
+```json
+{
+  "TokenProvider": {
+    "ClientId": "",
+    "ClientSecret": "",
+    "AuthorizationScope": ""
+  },
+  "EmailOptions": {
+    "SubscriptionId": ""
+  }
+}
+```
+
+## Upload attachment
+
+A [simple console application](https://github.com/kmdlogic/kmd-logic-email-client/tree/dev/sample/Kmd.Logic.Email.Client.AttachmentsSample) is included to demonstrate how to call Logic Emai API to `upload attachment file`. You will need to provide the settings described above in `appsettings.json`.
```

Comment:
I don't think you need to keep hyperlink for `simple` also. It can be for `console application` only.

Reply:
Resolved